blog/views.py

```diff
@@ -1,97 +1,22 @@
+from bootstrap.views import DeleteView
+from bootstrap.views import ListView
+from django.views.generic.detail import DetailView
 
-from django.core.paginator import EmptyPage
-from django.core.paginator import PageNotAnInteger
-from django.core.paginator import Paginator
-from django.shortcuts import render_to_response, get_object_or_404
-from django.views.generic.edit import FormView
-from django.views.generic.list import ListView
-
-from blog.forms  import ContactForm
-from blog.models import Category
-from blog.models import Contacts
 from blog.models import Entry
 
 
-class EntryView(ListView):
-    template_name = 'blog/blog.html'
+class EntryView(DetailView):
+    template_name = 'blog/view_entry.html'
     model = Entry
+    pk_url_kwarg = 'entry_id'
 
+class EntryDeleteView(DeleteView):
```

Comment:
Add the ``LoginRequiredViewMixin`` before the DeleteView.  You don't want just anyone to be able to delete your blog entries.